Jenkinsfile

```diff
@@ -14,7 +14,7 @@ parallel linux: {
         }
         stage('Build') {
             timeout(30) {
-                sh 'mvn -B -ntp -Dset.changelist clean install -Dmaven.test.failure.ignore -s settings-azure.xml -e'
```

Comment:
So delete `settings-azure.xml`. But first you would need to call `retrieveMavenSettingsFile`. And that requires `pwd tmp: true`.

Reply:
There seem to be required remaining pieces of @jglick's comment.